Can't resolve 'graphiql-subscriptions-fetcher'

I'm trying to build [graphiql-workspace-app](https://gitlab.com/kachkaev/graphiql-workspace-app) after updating to 1.0.5 and am getting this error:

```
Creating an optimized production build...
Failed to compile.

Module not found: Error: Can't resolve 'graphiql-subscriptions-fetcher' in '/path/to/graphiql-workspace-app/node_modules/graphiql-workspace/dist'
```

@tlvenn could this be related to your [changes in package.json](https://github.com/OlegIlyenko/graphiql-workspace/blame/master/package.json#L27-L29)?

__UPD:__ See [CI build logs](https://gitlab.com/kachkaev/graphiql-workspace-app/-/jobs/34422226).

--- a/package.json
+++ b/package.json
@@ -24,9 +24,6 @@
     "react": "global:React",
     "react-dom": "global:ReactDOM"
   },
-  "browser": {
-    "graphiql-subscriptions-fetcher": "./node_modules/graphiql-subscriptions-fetcher/dist/fetcher.js"
-  },
   "scripts": {
     "build": "./scripts/build.sh",
     "quickBuild": "./scripts/quickBuild.sh",

--- a/src/GraphiQLTab.jsx
+++ b/src/GraphiQLTab.jsx
@@ -24,7 +24,7 @@ import {introspectionQuery} from './utility/introspectionQueries';
 
 import {buildClientSchema} from 'graphql';
 
-import {graphQLFetcher} from 'graphiql-subscriptions-fetcher';
+import {graphQLFetcher} from 'graphiql-subscriptions-fetcher/dist/fetcher';
 import {SubscriptionClient} from 'subscriptions-transport-ws';
 
 import _ from 'lodash'

